spec/web/views/quizzes/index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require_relative '../../../../apps/web/views/quizzes/index'
+
+describe Web::Views::Quizzes::Index do
+  let(:exposures) { Hash[foo: 'bar'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.